src/game_battlealgorithm.cpp

```diff
@@ -641,7 +641,7 @@ bool Game_BattleAlgorithm::AlgorithmBase::IsTargetValid() const {
 		return false;
 	}
 
-	return (!GetTarget()->IsDead());
+	return GetTarget()->Exists();
```

Comment:
Commit message is bogus here.

7006d47b4bc

Reply:
Fixed